apps/portal/i18n/locales/it.json

```
@@ -0,0 +1,779 @@
+    "itemRestored": "Elemento ripristinato con successo.",
+    "problemRestoring": "Si è verificato un problema durante il ripristino",
+    "folderExists": "Questa cartella esiste già",
+    "folderNameTooLong": "Il nome della cartella non può contenere più di 35 caratteri",
+    "folderNameEmpty": "Il nome della cartella non può essere vuoto",
+    "enterFolderName": "Inserisci il nome della cartella",
+    "imageDownloaded": "Immagine scaricata",
+    "imageAdded": "Immagine aggiunta a: ",
+    "copied": "Copiato",
+    "videoMovedToTrash": "Il video è stato spostato nel Cestino",
+    "videoDownloaded": "Video scaricato",
+    "videoAdded": "Video aggiunto a:",
```

Biome (1.9.4)

[error] The key videoDownloaded was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)